test/TEST-85-SOCKET-PASS-FDS/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
No need to add a whole new test for this. Please just add a subtest in TEST-07-PID1.

Reply:
Will do. Thanks for the hint.